website/docs/r/project_service_identity.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_project_service_identity"
+sidebar_current: "docs-google-project-service-identity"
+description: |-
+ Generate service identity for a service.
+---
+
+# google\_project\_service\_identity
+
+~> **Warning:** These resources are in beta, and should be used with the terraform-provider-google-beta provider.
+See [Provider Versions](https://terraform.io/docs/providers/google/guides/provider_versions.html) for more details on beta resources.
+
+Generate service identity for a service.
+
+~> **Note**: Once created, this resource cannot be updated or destroyed. These
+actions are a no-op.
+
+To get more information about Service Identity, see:
+
+* [API documentation](https://cloud.google.com/service-usage/docs/reference/rest/v1beta1/services/generateServiceIdentity)
+
+<div class = "oics-button" style="float: right; margin: 0 0 -15px">
+  <a href="https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fterraform-google-modules%2Fdocs-examples.git&cloudshell_working_dir=healthcare_dataset_basic&cloudshell_image=gcr.io%2Fgraphite-cloud-shell-images%2Fterraform%3Alatest&open_in_editor=main.tf&cloudshell_print=.%2Fmotd&cloudshell_tutorial=.%2Ftutorial.md" target="_blank">
+    <img alt="Open in Cloud Shell" src="//gstatic.com/cloudssh/images/open-btn.svg" style="max-height: 44px; margin: 32px auto; max-width: 100%;">
+  </a>
+</div>
+## Example Usage - Service Identity Basic
+
+
+```hcl
+resource "google_project_service_identity" "hc_sa" {
+  service = "healthcare.googleapis.com"
+}
+
+resource "google_project_iam_member" "hc_sa_bq_jobuser" {
+    project = google_project_service_identity.hc_sa.project
+    role    = "roles/bigquery.jobUser"
+    member  = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-healthcare.iam.gserviceaccount.com"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `service` -
+  (Required)
+  The service to generate identity for.
+
+
+- - -
+
+* `project` - (Optional) The ID of the project in which the resource belongs.
+    If it is not provided, the provider project is used.
+
+
+## Timeouts
+
+This resource provides the following
+[Timeouts](/docs/configuration/resources.html#timeouts) configuration options:
+
+- `create` - Default is 4 minutes.
```

Comment:
These are each 20 minutes in the resource schema. Should we drop it here or there?

Reply:
Good catch, made it 20 and also removed delete since we mention deletion is a no-op.